plugins/src/opengrok/auth/plugin/ldap/LdapServer.java

```diff
@@ -176,7 +176,7 @@ private synchronized LdapContext connect() {
      * @param cons controls for the LDAP request
      * @return LDAP enumeration with the results
      *
-     * @throws NamingException
+     * @throws NamingException naming exception
```

Comment:
This kind of javadoc comments is redundant. I assume it was done to remove warnings while generating javadoc? If this is part of the public API then more reasonable comment is preferable. If not then I would remove the whole line.

Reply:
yep, it was sort of a desperate effort to clear the errors. I don't think the line can be removed.